src/mammos_entity/io.py

```diff
@@ -105,8 +122,13 @@ def entities_to_csv(
     dataframe = (
         pd.DataFrame(data, index=[0]) if all(if_scalar_list) else pd.DataFrame(data)
     )
-    with open(filename, "w") as f:
+    with open(_filename, "w") as f:
         f.write("#mammos csv v1\n")
```

Comment:
The version must be changed. Before you do anything: @samjrholt what do we now to with the version numbers `v1.1` or `v2`?

Reply:
When we originally discussed Hans, we said only using major versions. I don't have strong feelings but I am happy to continue with that.

Reply:
Also we never used `v1.0` so that implies that there are no minor versions